sonar-jproperties-plugin/src/main/java/org/sonar/plugins/jproperties/issuesaver/crossfile/DuplicatedKeysAcrossFilesCheckIssueSaver.java

```diff
@@ -26,46 +26,37 @@
 
 import org.sonar.jproperties.checks.generic.DuplicatedKeysAcrossFilesCheck;
 import org.sonar.jproperties.checks.generic.FileKeyTree;
-import org.sonar.plugins.jproperties.api.JavaPropertiesCheck;
 import org.sonar.plugins.jproperties.api.visitors.issue.IssueLocation;
 import org.sonar.plugins.jproperties.api.visitors.issue.PreciseIssue;
+import org.sonar.plugins.jproperties.issuesaver.IssueSaver;
 
-public class DuplicatedKeysAcrossFilesIssueSaver {
+public class DuplicatedKeysAcrossFilesCheckIssueSaver extends CrossFileCheckIssueSaver {
 
-  private DuplicatedKeysAcrossFilesIssueSaver() {
+  public DuplicatedKeysAcrossFilesCheckIssueSaver(IssueSaver issueSaver) {
+    super(issueSaver);
   }
 
-  public static void saveIssues(IssueSaver issueSaver) {
-    DuplicatedKeysAcrossFilesCheck check = null;
+  public void saveIssues() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)